README.md

```diff
@@ -4,8 +4,8 @@
 
 Blueprint is a React UI toolkit for the Web.
 
-Palantir uses Blueprint to build complex, data-dense web interfaces for _desktop applications_.
-If you rely heavily on mobile interactions and are looking for a mobile-first UI kit, this may not be for you.
+It is optimized for building complex, data-dense Web interfaces for _desktop applications_.
+If you rely heavily on mobile interactions and are looking for a mobile-first solution, this may not be for you.
```

Comment:
Word. How about "UI framework"? I know we use "React UI toolkit" in the tagline, but as a designer I find "UI kit" super overloaded https://dribbble.com/search?q=ui+kit